src/enums/events.js

```diff
@@ -101,6 +101,7 @@ const Event = {
     POSITION_PREPARE: 'position-prepare',
     POSITION_FULFIL: 'position-fulfil',
     PREPARE: 'prepare',
+    PREPARE_FORWARDED: 'prepare-forwarded',
```

Comment:
Do we need to call it prepare-forwarded? I think when we use this with event 'prepare', may be we can just call it 'forwarded'. So Event is `prepare` and action is `forwarded`

Reply:
Sounds good. done.